packages/user_packages/iotcc_mqtt_edge_system_stats.py

```diff
@@ -65,7 +65,14 @@ def read_cpu_utilization(sample_duration_sec=1):
 
 
 def read_disk_usage_stats():
-    return round(disk_stat.disk_reads_writes(disk_name)[0], 2)
+    # If the device raises an intermittent exception during metric collection process it will be required
```

Comment:
If the device raises an intermittent exception during metric collection process, user code will be required to handled it. Otherwise, if an Exception is thrown from UDM, collection process will be stopped for that metric.
 
If UDM returns a None value, then metric value for that particular collector instance won't be published.

Reply:
Fixed